charactersheet/charactersheet/viewmodels/character/status_line.js

```diff
@@ -34,7 +34,11 @@ function StatusLineViewModel() {
         if (!profile || statuses.length == 0) { return ''; }
 
         return profile.characterName() + ' is ' + statuses.map(function(e, i, _) {
-            return '<span class="text-' + e.type() + '">' + e.name() + '</span>';
-        }).join('&nbsp;,') + '.';
+            if (statuses.length > 1 && i == statuses.length - 1) {
+                return 'and <span class="text-' + e.type() + '">' + e.name() + '</span>';
```

Comment:
Not sure if this is the best way construct the sentence because it ends up in a weird looking sentence, like the one from your example; `, and inspired.`
Maybe generate the HTML beforehand and assign it as a property to the status model. This way, you can use the system you have now without making weird sentences. Thoughts?